docker-compose.yml

```diff
@@ -17,6 +17,12 @@ services:
       interval: 30s
       timeout: 5s
       retries: 2
+    user: "998:998"
```

Comment:
#763 would introduce an unprivileged user into the container and would make this option unnecessary. Perhaps wait for that?

Reply:
> #763 would introduce an unprivileged user into the container and would make this option unnecessary. Perhaps wait for that?

It would still be nice to force a UID/GID in the docker-compose file rather than trusting the container though?

Reply:
Because the point is reducing trust in the container?

Reply:
You sure? Because I would be nice to account for the event of the UID/GID inside the container being changed for whatever reason - malicious docker registry, malicious maintainer, etc.

Reply:
I don't see how it makes less sense. This is just generally limiting the container's permissions to protect the host.

This is analogous to me saying "Let's apply sandboxing onto X app" and you are saying "Let's just trust X app to sandbox itself, and if you don't trust it to do so then compile it yourself".